### DISCUSSION TITLE
New to Typo3 - New to Events

### DISCUSSION THREAD

**Discussion by dermaxem** [derhansen/sf_event_mgt#1030] [Q&A]
Hi there, i try to have a look for a problem on a fresh typo3 (version 11.5.14) installation.
"Got a Error if i click on " Events" in a fresh typo3-enviroment. :"Unable to find TypoScript settings for module."
Iam new to typo3 and cant find a point for this error. any hints?

thank thee.

**Comment by derhansen** [comment]
You missed step 2 of the installation manual - see https://docs.typo3.org/p/derhansen/sf_event_mgt/6.2/en-us/ForAdministrators/Installation/Index.html

Make sure to include the extensions TypoScript configuration in your own TypoScript template

![image](https://user-images.githubusercontent.com/2629896/187182179-5eb9beca-05da-4230-8683-d3bd9f717d8f.png)

Also make sure, that the system folder with your event records is below the TYPO3 root page (the one with the globe-icon) and that your TypoScript template is defined on the root page.
